js/client/bootstrap/modules/internal.js

```diff
@@ -137,19 +137,6 @@ let appendHeaders = function(appender, headers) {
     throw 'not connected';
   };
 
-  // //////////////////////////////////////////////////////////////////////////////
-  // / @brief rebuilds the authentication cache
-  // //////////////////////////////////////////////////////////////////////////////
-
-  exports.reloadAuth = function () {
```

Comment:
We are herewith removing `internal.reloadAuth`.
We need to put this into the list of incompatible changes at least. For 3.4, we could also leave the function in this module and make it a no-op, i.e. `exports.reloadAuth = function () {};`, deprecate it and document the change.

Reply:
IMHO there was no server implementation for that route, so that function was not working anyway